frontend/app/services/MemberService.scala

```diff
@@ -274,15 +274,19 @@ class MemberService(identityService: IdentityService,
     )).run
   }
 
-  def subscriptionUpgradableTo(sub: Subscription[SubscriptionPlan.Member], newTier: PaidTier): Boolean = {
-    import model.TierOrdering.upgradeOrdering
+  def upgradeTierIsValidForCurrency(sub: Subscription[SubscriptionPlan.Member], newTier: PaidTier): Boolean = {
     val newPlan = catalog.findPaid(newTier)
-
     // The year and month plans are guaranteed to have the same currencies
     val targetCurrencies = newPlan.year.charges.price.prices.map(_.currency).toSet
     val currencyIsAvailable = targetCurrencies.contains(sub.plan.currency)
+    currencyIsAvailable
```

Comment:
don't need to put into a val and then return it (although it is allowed if you think that's clearer that way)